### DISCUSSION TITLE
SF mtexColorbar range value

### DISCUSSION THREAD

**Discussion by lilarou** [mtex-toolbox/mtex#1736] [Ask Anything] [answered]
Hi all,

I plot IPF of some grains from EBSD map according to the calculated Schmid factor value.
`sSHex = symmetrise([sSHexBa;sSHexPa;sSHexPya;sSHexPy1a;sSHexPy2a],'antipodal');

stress= [0 0 0; 0 0 0; 0 0 -1];
sigma = stressTensor(stress);
g=grains(ID).meanOrientation;
sS = grains(ID).meanOrientation.*sSHex(Syst);

SF = abs(sS.SchmidFactor(sigma));

figure
plotIPDF(g, SF, [xvector,yvector,zvector], 'MarkerSize', 10,'MarkerEdgeColor', 'black')
mtexColorbar('title','Schmid Factor') `

I get the following IPF
![L4_SF_slip](https://user-images.githubusercontent.com/123663516/217780370-18096267-c4a6-4bd1-8073-c8d5000dca72.jpg)

I'd like to set the mtexColorbar range value to  [0,5]
If I add `**CLim(gcm,[0,5])**;` or `plotIPDF(g, SF, [xvector,yvector,zvector], **'colorrange',[0 5],**'MarkerSize', 10,'MarkerEdgeColor', 'black')` to my code I get the following IPF
![Q](https://user-images.githubusercontent.com/123663516/217781192-ab121abd-9b37-417c-8b3c-7f540dd12e89.jpg)

The mtexcolorbar is ok but not the projection of the grain's SF. 
Does anyone has an idea to fix this issue ?

Thank you

**Comment by kilir** [comment, marked as answer]
Hi,
your values are in the range ~0.2 - 0.5. For the color range 0-5 it makes totally sense that the color related to your sf values is in the very blue range then.
Cheers,
Rüdiger
